src/plugin/cursus/Entity/Course.php

```diff
@@ -86,9 +86,14 @@ class Course extends AbstractTraining
     private $propagateRegistration = false;
 
     /**
-     * @ORM\Column(name="session_duration", nullable=false, type="float", options={"default" = 1})
+     * @ORM\Column(name="session_days", nullable=false, type="float", options={"default" = 1})
      */
-    private $defaultSessionDuration = 1;
+    private $defaultSessionDays = 1;
+
+    /**
+     * @ORM\Column(name="session_hours", nullable=false, type="float", options={"default" = 0})
+     */
+    private $defaultSessionHours = 0;
```

Comment:
Data must be kept in one column. Just store the duration in hours and migrate exiting data in the Migration.

Reply:
Do I have to make a class inherited from Updater ?

Reply:
To create an Updater, you'll need to : 
- Create a updater class : https://github.com/claroline/Claroline/blob/13.x/src/plugin/cursus/Installation/Updater/Updater130013.php#L12. It should be named with the target version (imo it will be included with the 13.0.24 so it should be named Updater130024).
- Declare the new updater as a service (don't forget the tag `claroline.platform.updater`) : https://github.com/claroline/Claroline/blob/13.x/src/plugin/cursus/Resources/config/services/updater.yml#L23.
- Register it in the AdditionalInstaller of the plugin : https://github.com/claroline/Claroline/blob/13.x/src/plugin/cursus/Installation/AdditionalInstaller.php#L17